### DISCUSSION TITLE
paperswithcode导入相关问题

### DISCUSSION THREAD

**Discussion by fantasyTj** [Future-Scholars/paperlib#264] [General]
我在paperlib的官方文档-快速开始-添加补充材料那一张图的右下角看见了"code"一栏显示了代码的链接. 
这需要怎么实现呢?(我发现在scraper中有paperswithcode相关的scraper, 但是不知道怎么使用)

**Comment by GeoffreyChen777** [comment]
开启paperwithcode 就不用管了，能搜到代码的自动就有了。没显示就是没搜到。

你也可以手动编辑一个论文，编辑界面选高级，然后手动输入仓库链接

**Reply by fantasyTj** [reply]
感谢作者的回答, 之前的问题还是没有问清楚, 我再描述得清楚一些:
我目前使用的是chrome浏览器, 以drag-GAN这篇论文为例: 我在paperswithcode网站上找到了其地址https://paperswithcode.com/paper/drag-your-gan-interactive-point-based, 然后点击Paperlib插件, 显示no available importer. 
我还尝试进入对应的arxiv界面https://arxiv.org/abs/2305.10973v1, 并且在下方"Code, Data, Media"栏点击了"Papers with Code"按钮, 这种方式能够下载, 但最后还是没有显示codes一栏

**Reply by GeoffreyChen777** [reply]
啊你不需要去这些网站，你就在paperlib里，导入的任何一个论文如果有代码能搜到，就会自动显示了，前提是你开启了paperwithcode的搜刮器。如果没有你可以开启，然后右键这个论文条目，从…搜索里面点击paperwithcode，如果搜不到，那就是他们的API没搜到，因为软件使用api从paper with code搜的，跟他们网站界面搜有些不同。极少数会有这个情况。

**Reply by GeoffreyChen777** [reply]
另外你这个论文，我猜是因为那个冒号的问题，没匹配上，我等下更新下搜刮器，然后再试试

**Comment by GeoffreyChen777** [comment]
我突然发现了，这是个 Bug！

我会尽快发一个版本来修复这个问题！

感谢发现！

**Comment by GeoffreyChen777** [comment]
新版本修复了这个问题，请更新之后，点击重新搜取metadata试试。

点击菜单那个刷新按钮就行。

**Reply by fantasyTj** [reply]
十分感谢!
